### ISSUE TITLE
Post build scripts is not always executing

### ISSUE THREAD

**Issue by pawlowskim** [juicycleff/flutter-unity-view-widget#756]
**Describe the bug**
XCodePostBuild.cs is not trigger each time after building. I causes missing FUW interfaces and app build fails.

**To Reproduce**
I don't know exact path, for sometimes it just stops executing.

**Expected behavior**
Post build should always triggers.

**Unity (please complete the following information):**
 - OS: iOS
 - Version 2022.2.2f1 (but it happens for each unity editor I tried)

**Additional context**
I don't know why it's not get triggered, I'm not familiar with unity (just need to integrate it with flutter). But small change to the `XCodePsotBuild.cs` (fe. adding and enter) triggers unity editor to rebuild something, and it start triggering after that. People says to remove and install FUW unitypackage from scratch when this occurs. I guess it works the same way. If we remove it and re-install it will work.

When it's note executed for example those two interfaces are not added:

```
// Added by https://github.com/juicycleff/flutter-unity-view-widget
@protocol UnityEventListener <NSObject>
- (void)onSceneLoaded:(NSString *)name buildIndex:(NSInteger *)bIndex loaded:(bool *)isLoaded valid:(bool *)IsValid;
- (void)onMessage:(NSString *)message;
@end
```

**Comment by timbotimbo** [comment]
I've had 2 people bring this up in discord, but I haven't been able to reproduce it on my mac myself.

Can you try applying the changes in [this commit](https://github.com/timbotimbo/flutter-unity-view-widget/commit/0a1f92f710e44760f547bbd89a983ab138462db8)?
I replaced the Unity OnPostBuild trigger with the hope that a manual function call won't be skipped.
Make sure to delete the ios/UnityLibrary folder before trying, to avoid any possible conflicts with previous exports.

**Comment by pawlowskim** [comment]
Well, this should work. I will give it a try when it occurs again. What I was doing, is basically have 2 sample unity application, and just build iOS (Release). I was switching the unity projects, and building again, without any clean up on the flutter app side.

**Comment by pawlowskim** [comment]
I'm not sure if changes in the script files in general fixed the issue or the PR changes you mentioned, but after applying them, the "Post build" part was triggered (confirmed by adding FUW interfaces to `UnityAppController.h`)

**Comment by pawlowskim** [comment]
@timbotimbo Could you please create a PR for this change? Maybe it will be included at some point.

**Comment by timbotimbo** [comment]
Here you go.

@juicycleff  Could you take a look at some pull requests? I'd like to create a new unitypackage with some of the fixes that have built up recently.

**Comment by timbotimbo** [comment]
Included in the latest unitypackage 2022.2.0.
